mobile/src/main/java/org/openhab/habdroid/background/BackgroundTasksManager.kt

```diff
@@ -252,27 +256,36 @@ class BackgroundTasksManager : BroadcastReceiver() {
         const val WORKER_TAG_VOICE_COMMAND = "voiceCommand"
         fun buildWorkerTagForServer(id: Int) = "server-id-$id"
 
+        private const val GADGETBRIDGE_ACTION_PREFIX = "nodomain.freeyourgadget.gadgetbridge."
+        private val GADGETBRIDGE_ACTIONS = listOf(
+            "${GADGETBRIDGE_ACTION_PREFIX}FellAsleep",
+            "${GADGETBRIDGE_ACTION_PREFIX}WokeUp",
+            "${GADGETBRIDGE_ACTION_PREFIX}StartNonWear"
+        )
+
         internal val KNOWN_KEYS = listOf(
             PrefKeys.SEND_ALARM_CLOCK,
             PrefKeys.SEND_PHONE_STATE,
             PrefKeys.SEND_BATTERY_LEVEL,
             PrefKeys.SEND_CHARGING_STATE,
             PrefKeys.SEND_WIFI_SSID,
-            PrefKeys.SEND_DND_MODE
+            PrefKeys.SEND_DND_MODE,
+            PrefKeys.SEND_GADGETBRIDGE
         )
         internal val KNOWN_PERIODIC_KEYS = listOf(
             PrefKeys.SEND_BATTERY_LEVEL,
             PrefKeys.SEND_CHARGING_STATE,
             PrefKeys.SEND_WIFI_SSID,
-            PrefKeys.SEND_DND_MODE
+            PrefKeys.SEND_DND_MODE,
+            PrefKeys.SEND_GADGETBRIDGE // Required for EventListenerService
```

Comment:
TODO: This causes the periodic workers to run, but `triggerPeriodicWork()` will never trigger an update for Gadgetbridge, because there's no intent as input.